go.mod

```diff
@@ -39,6 +39,7 @@ require (
 	helm.sh/helm/v3 v3.8.1
 	k8s.io/api v0.23.5
 	k8s.io/apimachinery v0.23.5
+	k8s.io/cli-runtime v0.23.4
```

Comment:
Where is this dependency being used? Could you please try `go mod tidy` and see if this goes away?

Reply:
done/ the dependency was removed in some of the cleanup